addons/point_of_sale/models/pos_config.py

```diff
@@ -81,9 +81,8 @@ def _default_sale_journal(self):
     def _default_invoice_journal(self):
         return self.env['account.journal'].search([('type', '=', 'sale'), ('company_id', '=', self.env.company.id)], limit=1)
 
-    def _get_default_journal_ids(self):
-        journals = self.env['account.journal'].search([('journal_user', '=', True), ('type', 'in', ['cash','bank']), ('company_id', '=', self.env.company.id)])
-        return journals or False
+    def _default_payment_methods(self):
+        return self.env['pos.payment.method'].search([('split_transactions', '=', False), ('company_id', '=', self.env.company.id)])
```

Comment:
why not split transactions?

Reply:
The split_transaction field is only available in debug mode so in my opinion, it is best to avoid it as default.